Item/Item.module.less

```diff
@@ -104,5 +106,13 @@
 				}
 			}
 		});
+
+		&:global(.spottable):active,
+		:global(.spottable):active & {
+			.applySkins({
+				background-color: @agate-item-active-bg-color;
+				border-image-source: none;
```

Comment:
This overrides the `border-image-source` for `.focus`. So the border disappears when pressing an item. There should be the bottom border when pressing the item.
[
![image](https://user-images.githubusercontent.com/4239873/82857964-7906ac00-9f4d-11ea-92d5-96638676f5a1.png)
](url)